package_managers/install_pkgs.bzl

```diff
@@ -28,23 +53,39 @@ rm -f /var/cache/ldconfig/aux-cache
 rm -f /var/cache/apt/pkgcache.bin
 touch /run/mount/utab""".format(tar=tar, installation_cleanup_commands=installation_cleanup_commands)
 
-def _impl(ctx):
-  installables_tar = ctx.file.installables_tar.path
+def _impl(ctx, image_tar=None, installables_tar=None, installation_cleanup_commands="", output_image_name="", output_tar=None):
```

Comment:
My only feed back would be to leave signature of _imp unchanged and a new function sat  _install with this signature.

_imp will call _install with the right parameters.

Reply:
I'm a little unsure about if this new function is necessary. _impl is currently only called inside this rule and the additional attributes exposed in the _impl header doesn't affect how _impl is called now as all those attributes have default values. Adding this wrapper seems a little bit redundant to me. This pattern of exposing the attributes, outputs and implementation is also used in the container_image rule here (https://github.com/bazelbuild/rules_docker/blob/master/container/image.bzl#L163).

Reply:
@tejal29 what do you think?